BaseFilter/sections/specific.txt

```diff
@@ -129,8 +129,11 @@ kimcartoon.me##div > div[id][style*="overflow: hidden"][style*="width: 300px;"][
 ! /(readcomiconline|kimcartoon|kiss-anime|9anime|gogoanime|toonova|animeflv|animeplus|goodanime|animewow|animenova|animetoon|mangapanda|memecenter|mangareader)\.[a-z]{2,4}\/(?!(assets|scripts|player)\/)[a-zA-Z0-9]{4,}[\s\S]a*\/[a-zA-Z0-9]{4,}[\s\S]*.js/$domain=mangapanda.com|memecenter.com|mangareader.net|animetoon.org|goodanime.co|animeplus.tv|animeflv.net|toonova.net|gogoanime.in|kimcartoon.me
 !
 ! https://github.com/AdguardTeam/AdguardFilters/pull/168292
+! TODO: Remove once extension v4.4 is out
+!#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android && !adguard_ext_safari && !ext_ublock)
```

Comment:
But why `!adguard_ext_safari`?
`header` can't be used in Safari and with CB for Android
https://adguard.com/kb/general/ad-filtering/create-own-filters/#basic-rules-basic-modifiers

So can be changed to use only in AdGuard for Safari/iOS and CB for Android.

Reply:
We have separate rules under `!+ PLATFORM(ios, ext_safari)`. Does the regex with `{n,m}` work for Safari?

Reply:
> Does the regex with {n,m} work for Safari?

No(

Reply:
That's the reason.